Mathlib/AlgebraicGeometry/Morphisms/FormallyUnramified.lean

```diff
@@ -0,0 +1,96 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.AlgebraicGeometry.Morphisms.RingHomProperties
+import Mathlib.AlgebraicGeometry.Morphisms.OpenImmersion
+import Mathlib.RingTheory.RingHom.Unramified
+
+/-!
+# Formally unramified morphisms
+
+A morphism of schemes `f : X ⟶ Y` is formally unramified if for each affine `U ⊆ Y` and
+`V ⊆ f ⁻¹' U`, The induced map `Γ(Y, U) ⟶ Γ(X, V)` is formally unramified.
+
+We show that these properties are local, and are stable under compositions and base change.
+
+-/
+
+
+noncomputable section
+
+open CategoryTheory CategoryTheory.Limits Opposite TopologicalSpace
+
+universe v u
+
+namespace AlgebraicGeometry
+
+variable {X Y : Scheme.{u}} (f : X ⟶ Y)
+
+/-- A morphism of schemes `f : X ⟶ Y` is locally of finite type if for each affine `U ⊆ Y` and
+`V ⊆ f ⁻¹' U`, The induced map `Γ(Y, U) ⟶ Γ(X, V)` is of finite type.
```

Comment:
Is this docstring correct?

Reply:
Of course not 😅. Thanks for pointing it out.